src/integrationTest/java/org/opensearch/test/framework/matcher/GetSettingsResponseContainsIndicesMatcher.java

```diff
@@ -32,7 +33,7 @@ class GetSettingsResponseContainsIndicesMatcher extends TypeSafeDiagnosingMatche
 	@Override
 	protected boolean matchesSafely(GetSettingsResponse response, Description mismatchDescription) {
 
-		final ImmutableOpenMap<String, Settings> indexToSettings = response.getIndexToSettings();
+		final Map<String, Settings> indexToSettings = response.getIndexToSettings();
 		for (String index : expectedIndices) {
 			if (!indexToSettings.containsKey(index)) {
 				mismatchDescription
```

Comment:
Below this line I think you need to change the call from `.keys()` to `.keySet()`